src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECKSTATUS = 'my-app/categories/CHECKSTATUS';
```

Comment:
- [ ] Kindly update the action type to 
```javascript 
const CHECKSTATUS = 'bookstore/categories/CHECKSTATUS';
```
to match the duck pattern guidelines.

Reply:
Thank you for pointing this out. I have updated it now.